Mathlib/Algebra/Ring/Defs.lean

```diff
@@ -320,25 +351,6 @@ end HasDistribNeg
 ### Rings
```

Comment:
This heading is now a bit stale

Reply:
It still looks useful to me: it says that now we are going to talk about properties of rings, while we were dealing with semirings above.